runtime/src/bucket_map_holder.rs

```diff
@@ -153,6 +153,12 @@ impl<T: IndexValue> BucketMapHolder<T> {
         result
     }
 
+    /// prepare for this to be dynamic if necessary
+    /// For example, maybe startup has a shorter age interval.
+    fn age_interval_ms(&self) -> u64 {
+        AGE_MS
+    }
```

Comment:
I know this would be a pain to do now, but these time-like functions and constants could be `Duration` instead of a `u64`. And then only at the use in `AtomicInterval` convert back to a `u64` (or wherever that conversion is needed). Then the type system comes to our rescue and we hopefully don't have ms/us/ns mixups.